recipes/mpir/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "2.7.2" %}
+
+package:
+  name: mpir
+  version: {{ version }}
+
+source:
+  fn: mpir-{{ version }}.tar.bz2
+  url: http://mpir.org/mpir-{{ version }}.tar.bz2
+
+build:
+  number: 0
```

Comment:
Sorry, I think I spoke incorrectly as I see a `build.sh` file above. Can this be built on Mac and Linux?